tunnel/defaultroutemonitor.go

```diff
@@ -51,12 +51,14 @@ func bindSocketRoute(family winipcfg.AddressFamily, device *device.Device, ourLU
 	*lastLUID = luid
 	*lastIndex = index
 	blackhole := blackholeWhenLoop && index == 0
-	if family == windows.AF_INET {
-		log.Printf("Binding v4 socket to interface %d (blackhole=%v)", index, blackhole)
-		return device.BindSocketToInterface4(index, blackhole)
-	} else if family == windows.AF_INET6 {
-		log.Printf("Binding v6 socket to interface %d (blackhole=%v)", index, blackhole)
-		return device.BindSocketToInterface6(index, blackhole)
+	if bind, _ := device.Bind().(conn.BindSocketToInterface); bind != nil {
```

Comment:
This should return an error if it fails.

Reply:
Done.